pcl.orogen

```diff
@@ -1,7 +1,8 @@
 name "pcl"
 version "0.1"
 
-using_library 'pcl_common-1.7'
+pcl_ver = %x{dpkg -s libpcl-dev | grep Version}.match(/\d.\d/)[0]
```

Comment:
Hey. Thanks for taking care of this problem. However, you should definitely should not be using dpkg for this - it will fail if e.g. someone is using PCL from source. You can figure it out using `Utilrb::PkgConfig.each_package`, like so (untested).

~~~
if (pcl_available = Utilrb::PkgConfig.each_package(/^pcl_common-/).first)
  pcl_ver = available_pcl_common.match((\d+\.\d+)$/)[0]
else
  raise "cannot find the pcl_common pkg-config package"
end
~~~

Reply:
https://www.rubydoc.info/github/orocos-toolchain/utilrb/master/Utilrb/PkgConfig#each_package-class_method

Reply:
> The regex param does not seem to work. It always returns all available packages.

????

~~~
irb(main):004:0> Utilrb::PkgConfig.each_package(/test/).to_a
=> []
irb(main):005:0> Utilrb::PkgConfig.each_package(/cv/).to_a
=> ["opencv"]
irb(main):006:0>
~~~